scaling_performance/network_optimization.adoc

```diff
@@ -170,6 +170,31 @@ network infrastructure.
 VXLAN-offload does not reduce latency. However, CPU utilization is reduced even
 in latency tests.
 
+[[scaling-performance-optimizing-mtu]]
+=== Optimizing the MTU for Your Network
+
+There are two MTUs (maximum transmission units) that matter: the network interface card (NIC) MTU, and the SDN overlay's MTU.
+
+The NIC MTU can be whatever you think is optimal for your network.
```

Comment:
These look like the words from my email :)  Which is fine, I should have been more precise.  In general, setting the MTU for your overall network is pretty well covered by other resources, and what ben/jeremy have said.